stylesheet/styles.css

```diff
@@ -0,0 +1,276 @@
+  margin-right: 2em;
+  margin-top: 2em;
+}
+ul {
+  margin: 0;
+  padding: 0;
+
+}
+nav a {
+  text-decoration: none;
+  font-size: 16px;
+  color: black;
+  line-height: 2em;
+}
+nav a:hover {
+  text-decoration: underline;
+}
+nav li {
+  list-style: none;
+  float: left;
+  display: inline;
+  padding: 0px 20px;
+}
+nav {
+  font-size: 1rem;
+  background-color: #e7e7e7;
+  height: 3rem;
+  margin-top: 12rem;
+}
+nav ul, {
+  margin: 0 auto;
+  width: 70em;
+
+}
+main {
+  width: 80%;
+  margin: 0 auto;
+}
+/* Content Container */
+#content-container {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+  margin: 1em 0;
+  padding: 0 .5em;
+
+}
+.content p {
+  padding-left: 2em;
+  size: 2em;
+}
+.content h1 {
+  size: 4em;
+  padding-left: 1em;
+}
+
+.content-photo {
+  width: 20%;
+  margin-left: 2%;
+}
+.content-text {
+  width: 50%;
+  margin-left: 2%;
+}
+/* Adopt Container */
+#adopt-container {
+  display: flex;
+  flex-flow: wrap;
+  justify-content: space-between;
+  width: 100%;
+  margin: 1em 0;
+}
+.adopt-item-1 {
+  width: 23%;
+  border: 2px grey solid;
+  margin-bottom: 2%;
+  padding: 1em;
+}
+.adopt-item-1 > img {
+  width: 100%;
+  cursor: pointer;
+  border: 2px white solid;
+}
+.adopt-item-1 > img:hover {
+  border: 2px green solid;
+}
+.adopt-item-1 > h1 {
+  color: blue;
+}
+.adopt-btn > .button {
+  border: 2px white solid;
+  padding: 2% 20%;
+  background-color: green;
+  color: white;
+  text-decoration: none;
+  text-align: center;
+  font-size: 1rem;
+  width: 100%;
+}
+.adopt-btn > .button:hover {
+  border: 2px red solid;
+  background-color: black;
+  transition: all 0.5s ease;
+  color: red;
+  cursor: pointer;
+
+}
+/* Blog Container */
+#blog-container {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: wrap;
+  flex-direction: column;
+}
+.blog-image {
+  width: 25%;
+}
+.blog-text {
+  width: 70%;
+}
+/* Adopt More btn */
+.adopt-more {
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+  padding: 8px 30px;
+  color: white;
+  text-decoration: none;
+  text-align: center;
+}
+.adopt-more #more-btn {
+  background-color: grey;
+  color: white;
+  width: 160px;
+  padding: 2px 20px;
+  height: 35px;
+  font-size: .8em;
+}
+.adopt-more #more-btn:hover {
+  background-color: green;
+  transition: all 0.5s ease;
+  cursor: pointer;
+  border: 2px red solid;
+  color: red;
+}
+/* Footer */
+footer {
+  text-align: center;
+  height: 300px;
+  background-color: blue;
+}
+.ft-container {
+  margin: 0 auto;
+  width: 40em;
+  color: white;
+}
+.ft-nav {
+  color: white;
+  font-size: 1em;
+}
+.ft-nav li {
+  list-style: none;
+  display: inline;
+  padding: 0px 10px;
+  text-decoration: none;
+}
+.ft-nav a {
+  text-decoration: none;
+  font-size: 16px;
+  color: white;
+  line-height: 2em;
+}
+@media screen and (max-width:1024px) {
+  #content-container, #adopt-container,
+  header, .content text, .ft-nav {
+    width: 100%;
+  }
+  .adopt-item-1 {
```

Comment:
we'll talk about media queries later, but I'd aim to have 4 wide on desktop, three wide on tablet and 1 wide on mobile.